README.md

```diff
@@ -61,8 +60,28 @@ Note: If you have cloned the repo previously make sure you remove your node_modu
 - `./gradlew publishAndroid`
 - The compiled version of the Android module is here: `<project-root>/android`
 
+Note: On Windows the RealmJS repo should be cloned with symlynks enabled 
+```
+#run in elevated command prompt
+git clone -c core.symlinks=true https://github.com/realm/realm-js
+
+```
+
+or manually create the symlinks using directory junctions
+```
+#run in elevated command prompt
+cd realm-js\react-native\android\src\main\jni
+mklink /j "src" "../../../../../src/"
+mklink /j "vendor" "../../../../../vendor"
+cd realm-js\tests\react-test-app\android\app\src\main
+mklink /j assets "../../../../../data"
+```
+
 ### Building for nodejs:
 Be sure you have python2.7 as the default python. 3.x won't work, and it's not enough to use `--python=python2.7` as parameter to npm.
+```
+brew install python@2
```

Comment:
Are we expecting everyone to use brew?

Reply:
brew is already mentioned in the readme before this mention. 
https://github.com/realm/realm-js/blob/5cf0c6d1121e73553dbc4f0696a5c1f503f3e816/README.md#L121

Reply:
It's after. Then I guess we should mention it as prerequisites in the top? Or mention that's just an example and people can install it the way they want?

Reply:
What I meant is it was mentioned in here before I added this change.
I guess it is just an example. Should we explicitly state this is an example how to install python?